aten/src/ATen/native/quantized/cpu/qnnpack/src/init.c

```diff
@@ -11,6 +11,9 @@
 #include <stdint.h>
 
 #ifdef _MSC_VER
+#ifndef WIN32_LEAN_AND_MEAN
```

Comment:
I don't know if I should import `c10/util/win32-headers.h` here. Same for the code in `torch` and `modules`. So I just use `WIN32_LEAN_AND_MEAN` instead.

Reply:
I think we should not use it here. Because qnnpack is a standalone library.